tests/type_propagation/gold/default-arguments-test.gold-O2

```diff
@@ -84,6 +84,4 @@ test-non-default tests/type_propagation/default-arguments-test.toit
 test-non-default-non-literal tests/type_propagation/default-arguments-test.toit
  - argument 0: {True|False}
   0[016] - load local 2
```

Comment:
Not entirely sure why this local doesn't go away. Checking.

Reply:
Oh, wait. We're returning the local. It's the load local 2 from position 4 that sticks around. All is well :)